Automattic/wp-calypso
# Themes: Have Support link point to new Details page, rename to Setup

Also fixes a bug that had the Support link disabled in single-site mode.

To test:
* Check that there are 'Setup' links for premium themes in single-site, multi-site, and logged-out mode, and that they correctly point to the new theme details sheet (e.g. http://calypso.localhost:3000/theme/passenger/setup/<mysite>).
* Also check that in single-site mode, the current theme bar has a 'Setup' button, again pointing to the correct destination.
* In `development.json`, disable `manage/themes/details`, and do the same checks again -- this time, the 'Setup' links should point to the old showcase, i.e. `http://wordpress.com/themes/<mysite>/rosalie/support`
